buildSrc/src/main/java/Dependencies.kt

```diff
@@ -91,7 +94,7 @@ object Deps {
     const val mozilla_ui_icons = "org.mozilla.components:ui-icons:${Versions.mozilla_android_components}"
 
     const val mozilla_service_firefox_accounts = "org.mozilla.components:service-firefox-accounts:${Versions.mozilla_android_components}"
-    const val mozilla_service_glean = "org.mozilla.components:service-glean:${Versions.mozilla_android_components}"
+    const val mozilla_service_glean = "org.mozilla.components:service-glean:${Versions.mozilla_service_glean}"
```

Comment:
Why is a different glean version needed here?

Reply:
While I was depending directly to the megazord-forUnitTests, I was also depending on the org.mozilla.telemetry:glean-forUnitTests that is stuck to version 25.0.1. I wrongly used that version also for org.mozilla.components:service-glean. Corrected.